src/App/Header/UserMenu/Menu.tsx

```diff
@@ -11,15 +10,13 @@ type Props = {
   signOut(): void;
   user: AuthenticatedUser;
 };
-export default function Menu({ classes = "", user, signOut }: Props) {
+export default function Menu({ user, signOut }: Props) {
   return (
-    <Popover.Panel
-      className={`${classes} shadow-xl bg-gray-l6 dark:bg-blue-d6 w-max rounded overflow-hidden`}
-    >
+    <>
+      <hr className="my-3 text-navy-l1" />
```

Comment:
Can use [src/components/Separator.tsx](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9.2/src/components/Separator.tsx) to keep separators aligned throughout the app, like `<Separator classes="my-3" />`: 
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/71575689-5e38-44dd-bb0a-f5e44a20eab5)

Reply:
I used an hr here since the App Footer has it's own Separator defined in there. Will look into using that component.

Reply:
Fixed in [818709c](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2815/commits/818709c7189891fd4e06b98ebd763c1216d94fae)